projects/go-style-guide/src/app/features/ui-kit/components/form-docs/components/select-docs/select-docs.component.ts

```diff
@@ -207,7 +207,8 @@ export class SelectDocsComponent implements OnInit {
     [items]="options$ | async"
     [multiple]="true"
     label="Your Input"
-    [typeahead]="itemInput">
+    [typeahead]="itemInput"
+    [typeToSearchText]="'Search The Thing'">
```

Comment:
same thing here `typeToSearchText="Search The Thing"`

Reply:
Also switched this.